ocaml/xapi/cli_operations.ml

```diff
@@ -3249,16 +3254,15 @@ let with_license_server_changes printer rpc session_id params hosts f =
 
 let host_apply_edition printer rpc session_id params =
   let host =
-    if List.mem_assoc "host-uuid" params then
-      Client.Host.get_by_uuid rpc session_id (List.assoc "host-uuid" params)
-    else
-      get_host_from_session rpc session_id in
+    if List.mem_assoc "host-uuid" params
+    then Client.Host.get_by_uuid rpc session_id (List.assoc "host-uuid" params)
```

Comment:
This seems an unnecessary change?

Reply:
I may have gone on a slight spree in order to make some code more readable (have since moved that to a separate branch), though if/then/else just comes down to preference. Should I remove it, or just bundle it into a separate PR?

Reply:
I happen to think that the original code is more readable ;)

Generally speaking, we should avoid making arbitrary style changes unless your changing those lines for another reason as well, or the new version is obviously Much Better™.